drafthorse/models/party.py

```diff
@@ -59,6 +59,22 @@ class Meta:
         tag = "EmailURIUniversalCommunication"
 
 
+class LegalOrganization(Element):
+    id = StringField(NS_RAM, "ID", required=False, profile=BASIC)
```

Comment:
Maybe this should be an IDField as it reads type="udt:IDType" in the spec which allows for an optional schemeID. There are other places where StringFields might need to be promoted to IDFields.